fearless/Common/ViewController/SheetAlertViewLayout.swift

```diff
@@ -89,6 +91,26 @@ final class SheetAlertViewLayout: UIView {
         }
     }
 
+    private func descriptionLabelLayoutSubviews() {
+        guard let descriptionText = descriptionLabel.text else {
+            return
+        }
+        let labelFullHeight = descriptionText.height(withConstrainedWidth: frame.width, font: descriptionLabel.font)
+        let viewHeight = bounds.height
+        let screenHeight = UIScreen.main.bounds.height
+
+        if (labelFullHeight + viewHeight) > screenHeight * 0.7 {
```

Comment:
please create variable for 0.7 consant and name it like 'popupWindowHeightRatio'